### ISSUE TITLE
Error when deleting bean with primary key on multiple columns

### ISSUE THREAD

**Issue by Zheness** [thecodingmachine/tdbm#93]
Hello,

I have a table `participation` with 3 columns:
- event_code (varchar)
- idcompany (int)
- key (varchar)

My primary key is event_code AND idcompany.

When I try to delete a participation I have this error:

```
Doctrine\DBAL\Exception\InvalidArgumentException: Empty criteria was used, expected non-empty criteria

/var/www/html/vendor/doctrine/dbal/lib/Doctrine/DBAL/Exception/InvalidArgumentException.php:38
/var/www/html/vendor/doctrine/dbal/lib/Doctrine/DBAL/Connection.php:675
/var/www/html/vendor/thecodingmachine/tdbm/src/TDBMService.php:287
```

Is it a problem on my side ?

Thanks

**Comment by moufmouf** [comment]
It might be a problem with TDBM... I'll check this.

**Comment by moufmouf** [comment]
Looking at the issue, there seems to be a lot of problems when a primary key on multiple columns contains a column that is a foreign key...

Still looking into it.

**Comment by moufmouf** [comment]
Fixed by #102
